app/api/schema.py

```diff
@@ -1,5 +1,134 @@
-from graphene import AbstractType
+import graphene
+from graphene import relay, Field, String, Int, Boolean, List
+from graphene_django import DjangoConnectionField, DjangoObjectType
+from graphql_relay.node.node import from_global_id
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
 
 
-class Query(AbstractType):
-    pass
+def getErrors(e):
```

Comment:
@andela-aoduola: For clean organization, I suggest you create a `utils.py` module under this app and move this function there. Create a new package and move line 19-123 to a new module under it. So this way, subsequent graphql CRUD implementations for the remaining models will be in separate modules. This is just my own way of organizing it. Please, feel free to do it your way if you have one.